### DISCUSSION TITLE
Temp Dirs

### DISCUSSION THREAD

**Discussion by bmentink** [ericaltendorf/plotman#725] [General]
HI,

Love the software, make's plotting much easier. However ..

I am getting the error: "no eligible tempdirs (5617s/5100s)" when I plot.

I am only running two parallel plots with a 1TB SSD. I would have thought that since temporary storage takes 280GB per plot, there would be enough temp space ..

**Comment by byeuji** [comment]
If you're plotting in parallel or using the tmp2 feature, it might just be a temporary issue. What plotman probably means is "no eligible tempdirs *right now*"

You might also have the max jobs `tmpdir_max_jobs` setting too low -- with 1TB plotting drive, you should be able to set this to 4, since using plotman, not all jobs will be in Phase 2 at the same time. But personally, I'd recommend using the tmp2 feature of chia to avoid running out of space on tmp and gumming up your plots.
